tests/agent/message_service/test_subscriber.py

```diff
@@ -11,7 +11,7 @@
 )
 from tests.agent.message_service.amqp_errors import get_amqp_errors
 
-pytestmark = pytest.mark.unit
+pytestmark = [pytest.mark.unit, pytest.mark.cloud]
```

Comment:
These labels should be mutually exclusive so tests will only one in 1 stage. This happens below also.

Reply:
done